src/scenes/ObservationsRouterV2/ListView/ObservationMap.tsx

```diff
@@ -1,40 +1,81 @@
 import usePlantMarkersMapLegend from 'src/components/NewMap/usePlantMarkersMapLegend';
 import usePlantingSiteMapLegend from 'src/components/NewMap/usePlantingSiteMapLegend';
 import usePlotPhotosMapLegend from 'src/components/NewMap/usePlotPhotosMapLegend';
 import useSurvivalRateMapLegend from 'src/components/NewMap/useSurvivalRateMapLegend';
 import { getBoundingBoxFromPoints } from 'src/components/NewMap/utils';
 import { useOrganization } from 'src/providers';
-import { useLazyGetPlantingSiteQuery, useLazyListPlantingSitesQuery } from 'src/queries/generated/plantingSites';
+import {
+  ObservationMonitoringPlotResultsPayload,
+  useLazyListAdHocObservationResultsQuery,
+  useLazyListObservationResultsQuery,
+} from 'src/queries/generated/observations';
+import {
+  useLazyGetPlantingSiteHistoryQuery,
+  useLazyGetPlantingSiteQuery,
+  useLazyListPlantingSitesQuery,
+} from 'src/queries/generated/plantingSites';
+import {
+  ObservationMonitoringPlotPhoto,
+  ObservationMonitoringPlotPhotoWithGps,
+  RecordedPlantStatus,
+} from 'src/types/Observations';
 import useMapboxToken from 'src/utils/useMapboxToken';
 
 type ObservationMapProps = {
+  isBiomass?: boolean;
   plantingSiteId?: number;
   selectPlantingSiteId: (siteId: number) => void;
 };
 
-const ObservationMap = ({ plantingSiteId, selectPlantingSiteId }: ObservationMapProps) => {
+const ObservationMap = ({ isBiomass, plantingSiteId, selectPlantingSiteId }: ObservationMapProps) => {
   const { mapId, token } = useMapboxToken();
   const { selectedOrganization } = useOrganization();
   const mapRef = useRef<MapRef | null>(null);
   const { fitBounds } = useMapUtils(mapRef);
 
   const { selectedLayer, plantingSiteLegendGroup } = usePlantingSiteMapLegend('sites', plantingSiteId === undefined);
-  const { plantMakersLegendGroup } = usePlantMarkersMapLegend(plantingSiteId === undefined);
-  const { plotPhotosLegendGroup } = usePlotPhotosMapLegend(plantingSiteId === undefined);
-  const { survivalRateLegendGroup } = useSurvivalRateMapLegend(plantingSiteId === undefined);
+  const { livePlantsVisible, deadPlantsVisible, plantMakersLegendGroup } = usePlantMarkersMapLegend(
+    plantingSiteId === undefined
+  );
+  const { plotPhotosVisible, plotPhotosLegendGroup } = usePlotPhotosMapLegend(plantingSiteId === undefined);
+  const { survivalRateVisible, survivalRateLegendGroup } = useSurvivalRateMapLegend(plantingSiteId === undefined);
+  const { adHocPlotsVisible, permanentPlotsVisible, temporaryPlotsVisible, monitoringPlotsLegendGroup } =
+    useMonitoringPlotsMapLegend(plantingSiteId === undefined, isBiomass, isBiomass);
 
-  const { sitesLayerStyle, zonesLayerStyle, subzonesLayerStyle } = useMapFeatureStyles();
+  const {
+    sitesLayerStyle,
+    zonesLayerStyle,
+    subzonesLayerStyle,
+    adHocPlotsLayerStyle,
+    permanentPlotsLayerStyle,
+    temporaryPlotsLayerStyle,
+    survivalRate50To75,
+    survivalRateLessThan50,
+    survivalRateMoreThan75,
+  } = useMapFeatureStyles();
```

Comment:
Could also optionally not destructure since there's so many properties being retrieved, e.g.
```ts
const mapFeatureStyles = useMapFeatureStyles();
...
style: mapFeatureStyles.sitesLayerStyle,
```